src/cmd/tools/m3ctl/namespaces/types.go

```diff
@@ -22,7 +22,7 @@ package namespaces
 
 const (
 	// DefaultPath is the default url path for the namespace api calls
-	DefaultPath = "/api/v1/namespace"
+	DefaultPath = "/api/v1/services/m3db/namespace"
```

Comment:
What's the rationale for changing the URL path here? This is a breaking change API wise (not sure how many users depend on this URL path). Also in this PR too since it's for removing deprecated fields haha.

Reply:
I didn't make the actual call here, but @robskillington included this along w/ the changes to the deprecated fields I assume just because it is similarly breaking. I'd imagine we want to consider these routes deprecated as well.

Reply:
Yeah so this was an old URL that no one should be using anymore, the operator does not use it any longer, it was before the coordinator could actually manage multiple services.